src/PerfView/PerfViewData.cs

```diff
@@ -4120,9 +4121,54 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
                 pinnedObjectAnalyzer.Execute(GCPinnedObjectViewType.PinnedObjectAllocations);
             }
             else if (streamName == "CCW Ref Count")
-            {
-                // TODO use the callback model.  We seem to have an issue getting the names however. 
-                foreach (var data in events.ByEventType<CCWRefCountChangeTraceData>())
+            {                
+                eventSource.Interop.AddCallbackForEvents<TaskCCWQueryRuntimeClassNameArgs>(args =>
+                {
```

Comment:
Can you tell me more about when this fires?   Is it useful?

Reply:
@vancem I assume this event might be useful because it contains information about the namespace and class name of CCW type. does it make sense?

Reply:
I don't know enough about CCWQueryRuntimeClassName to understand if it is useful.   As you point out it might be because it has a name, but I suspect it won't be that useful because it fires only when the code actually calls a certain COM API (to get the class name), and that only happens rarely.

However even if the information is useful, we don't want it in this view because is throws off the ref count (we want increfCount to have a metric of 1 and decRefCount to have a metric of -1 so that we only see non-zero values for com objects that are still alive.   If you add in any other event with a non-zero metric you ruin that property.  

The fact that the Inc and Dec events have no way of getting a COM class name is unfortunate, and diminishes the value of the view but fixing that will require a change in CoreRT (which is not that big of a deal, we probably want to pursue it).  But that is a later step, for now we want to get the information we HAVE presented in a useful way (which means just using inc and dec).